flight/targets/flyingpio/flyingpi.cfg

```diff
@@ -0,0 +1,27 @@
+#
+# Config for programming FlyingPIO target from Raspberry Pi.
+#
+
+interface sysfsgpio
+#interface bcm2835gpio
+
+#bmc2835gpio_srst_num 17
+sysfsgpio_srst_num 17
+
+transport select swd
+
+sysfsgpio_swd_nums 18 27
+#bcm2835gpio_swd_nums 18 27
+
+source /usr/local/share/openocd/scripts/target/stm32f0x.cfg
```

Comment:
Unfortunately so is the rest.  We have to build openocd ourselves because raspibian distributed stuff doesn't support SWD in the sysfsgpio driver.   Will fix this though.

(Ultimately we will need our own debian-style ppa)